specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -74,18 +74,41 @@ export function TreeRow<SCHEMA extends AnyTree>({
   const isExpanded = Array.isArray(conformation);
   const isLoading = isExpanded && !Array.isArray(rows);
   const displayChildren = isExpanded && typeof rows?.[0] === 'object';
+  const orderByField = userPreferences.get(
+    'treeEditor',
+    'behavior',
+    'orderByField'
+  );
+
   React.useEffect(() => {
     if (!isLoading) return undefined;
 
-    void getRows(row.nodeId).then((rows) =>
-      destructorCalled ? undefined : setRows(rows)
-    );
+    void getRows(row.nodeId).then((fetchedRows: RA<Row>) => {
+      const sortedRows = Array.from(fetchedRows).sort((a, b) => {
+        switch (orderByField) {
+          case 'rankID': {
```

Comment:
in the UserDefinitions.tsx this was `rankId` but here is it checking (wrongly) for `rankID`

Reply:
[`7ef5726` (#6046)](https://github.com/specify/specify7/pull/6046/commits/7ef5726d741edac702588f3d5fd71c03448e82e8)